etc/ci/upload_nightly.sh

```diff
@@ -13,6 +13,7 @@ usage() {
 
 upload() {
     s3cmd put "${2}" "s3://servo-builds/nightly/${1}/"
+    s3cmd cp "s3://servo-builds/nightly/${1}/${2}" "s3://servo-builds/nightly/${1}/servo-latest.${3}"
```

Comment:
@aneeshusa Are you sure? Below is literally (from my history) how I created the servo-latest file last night:

```
[larsberg@larsberg foo]$ s3cmd cp s3://servo-builds/nightly/linux/2016-07-01T05:28:51-servo-tech-demo.tar.gz s3://servo-builds/nightly/linux/servo-latest.tar.gz
remote copy: 's3://servo-builds/nightly/linux/2016-07-01T05:28:51-servo-tech-demo.tar.gz' -> 's3://servo-builds/nightly/linux/servo-latest.tar.gz'
```